src/webui/api/authcontroller.h

```diff
@@ -47,12 +47,14 @@ class AuthController : public APIController
 
 private slots:
     void loginAction();
+    void authAction();
```

Comment:
IMO, having two actions/endpoints for the same thing (login) isn't a good idea.

Reply:
Changed to `token_mode` param in login action.